v4/core/utils.go

```diff
@@ -179,3 +179,57 @@ func PrettyPrint(result interface{}, resultName string) {
 func GetCurrentTime() int64 {
 	return time.Now().Unix()
 }
+
+// ConvertSlice Marshals 'slice' to a json string, performs
+// string manipulation on the resulting string, and converts
+// the string to a '[]string'. If 'slice' is nil, not a 'slice' type,
+// or an error occurred during conversion, an empty slice will be returned.
+func ConvertSlice(slice interface{}) (s []string) {
```

Comment:
I couldn't find a way to restrict this method to only accept a parameter that is a slice, so I had to make the input an ``interface{}`` then use reflection to validate if in fact it was a slice.